src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/internal/FontCache/FamilyCollection.cs

```diff
@@ -410,10 +409,12 @@ internal static FamilyCollection FromWindowsFonts(Uri folderUri)
                 }
 
                 // If there was styling information.
-                if (potentialFaceName.Length > 0)
+                if (familyName.Length != originalFamilyName.Length)
                 {
-                    // The first character in the potentialFaceName will be a space so we need to strip it off.
-                    Text.TextInterface.Font font = GetFontFromFamily(fontFamilyDWrite, potentialFaceName.ToString(1, potentialFaceName.Length - 1));
+                    // To obtain the face name, we remove the family name and the next char (A space) from the original family name.
+                    int faceNameIndex = familyName.Length + 1;
+                    ReadOnlySpan<char> faceName = originalFamilyName.AsSpan(faceNameIndex, originalFamilyName.Length - faceNameIndex);
```

Comment:
```suggestion
                    ReadOnlySpan<char> faceName = originalFamilyName.AsSpan(faceNameIndex);
```

Reply:
Thanks for the suggestion! I commited your suggested change